test/decoder_test.exs

```diff
@@ -0,0 +1,37 @@
+defmodule DecoderTest do
+  import Membrane.Testing.Pipeline.Assertions
+  alias Membrane.Pipeline
+  use ExUnit.Case
```

Comment:
Take a look at:
https://github.com/lexmag/elixir-style-guide#modules

We do not enforce it at all but in most modules we order them in similar way.